dune/gdt/local/bilinear-forms/integrals.hh

```diff
@@ -96,7 +96,9 @@ public:
     result *= 0;
     // loop over all quadrature points
     const size_t integrand_order = integrand_->order(test_basis, ansatz_basis) + over_integrate_;
-    for (const auto& quadrature_point : QuadratureRules<D, d>::rule(element.geometry().type(), integrand_order)) {
+    assert(integrand_order < std::numeric_limits<int>::max());
```

Comment:
This should not be required and is a left-over from the old master. All `order()` methods now return ints, so the `integrand_order` is already an `int`. Thus we neither need the assertion, nor the `static_cast`. (As a side note: we do have the `numeric_cast` in dune-xt-common to perform the check and the cast, and to give an error message on failure.) Long story short, the following code should suffice:
```c++
const auto integrand_order = integrand_->order(test_basis, ansatz_basis) + over_integrate_;
for (const auto& quadrature_point : QuadratureRules<D, d>::rule(element.geometry().type(), integrand_order)) {
// ...
```